cross-chain/optimism/.gitignore

```diff
@@ -0,0 +1,15 @@
+# Hardhat
+/artifacts/
+/build/
+/cache/
+/export/
+/external/npm
+/typechain/
+/export.json
+/deployments/*
+!/deployments/mainnet/
+!/deployments/optimisticEthereum/
```

Comment:
![Screenshot 2023-04-06 at 14 42 41](https://user-images.githubusercontent.com/3156420/230382060-dc4ee452-a17f-4aa4-873d-1ab06eded5bc.png)

Those are the names supported by `hardhat-etherscan`. I decided to align everything with just one name `optimistic`. Not having both `optimistic` and `optimism` as I thought this might be confusing.  "Optimistic" is also commonly used, e.g the official block explorer for this network is `https://optimistic.etherscan.io/`

However, I'm okay changing it to `OptimismWormholeGateway` and `OptimismTBTC`. Having just one name was my preference, but I agree that "Optimism" is known better.

Reply:
Agreed, it makes sense to use the same name everywhere but I think it makes more sense to use "optimism" as it is way more common. I wouldn't worry about `hardhat-etherscan`. I think they named it after `https://optimistic.etherscan.io/` but I do not know why they named the domain "optimistic", especially since the rendered page says "Powered by Optimism".

Reply:
Ok, renamed everything to "optimism" [e40483c](https://github.com/keep-network/tbtc-v2/pull/585/commits/e40483ca6436015c9858cd7e6ad99dd694d7a8dd)